platform-sdk/consensus-gossip-impl/src/main/java/module-info.java

```diff
@@ -0,0 +1,8 @@
+module com.hedera.consensus.gossip.impl {
+    exports com.hedera.service.gossip.impl;
```

Comment:
Do we need to export this? Or is implementing the GossipServiceImpl (`provides` clause below) sufficient?